File: Service/ClientCare/RMA.Service.ClientCare.Contracts/Entities/Member/UploadRatesSummary.cs
namespace RMA.Service.ClientCare.Contracts.Entities.Member
{
    public class UploadRatesSummary
    {
        public double Total { get; set; }
        public double TotalUploaded { get; set; }
        public double TotalSkipped { get; set; }
        public int ValidationCount { get; set; }
        public string FileIdentifier { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/clientcare/src/app/policy-manager/shared/entities/upload-rates-summary.ts
export class UploadRatesSummary {
  total: number;
  totalUploaded: number;
  totalSkipped: number;
  validationCount: number;
  fileIdentifier: string;
}

export class RatesUploadErrorAudit{
  id: number;
  fileIdentifier: string;
  fileName: string;
  errorCategory: string;
  errorMessage: string;
  excelRowNumber: string;
  uploadDate: Date;
}